src/test/java/com/wikia/webdriver/common/contentpatterns/URLsContent.java

```diff
@@ -172,6 +173,7 @@
 
   //External URL
   public static final String EXTERNAL_URL = "http://www.wikia.com";
+  public static final String EXTERNAL_RUNESCAPE = "http://runescape-test.wikia.com/";
```

Comment:
I think that until the flag will not be active on mediawiki119 it will not be tested on production wikii